plugins/modules/ldap_attrs.py

```diff
@@ -57,8 +57,12 @@
     required: true
     type: dict
     description:
-      - The attribute(s) and value(s) to add or remove. The complex argument format is required in order to pass
-        a list of strings (see examples).
+      - The attribute(s) and value(s) to add or remove.
+      - Each attribute value can be a string for single-valued attributes or
+        a list of strings for multi-valued attributes.
+        To break long string values
+        into multiple lines for readability use the YAML block modifiers as
+        seen in the examples.
```

Comment:
I would start this as a new paragraph, and then add more details about type conversion, something like:
```suggestion
      - To break long string values
        into multiple lines for readability use the YAML block modifiers as
        seen in the examples.
      - Note that when using values that YAML/ansible-core interprets as other types,
        like C(yes), C(no) (booleans), or C(2.10) (float), make sure to quote them if
        these are meant to be strings. Otherwise you risk that the wrong values are
        sent to LDAP.
```
Same in the other module.